packages/components/addon/components/hds/text/display.js

```diff
@@ -0,0 +1,74 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+// notice: only some combinations of size + font-weight are allowed (per design specs)
+// see: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=1262%3A9192
+
+export const DEFAULT_SIZE = '200';
+export const AVAILABLE_SIZES = [500, 400, 300, 200, 100];
+
+// notice: the first item in the array is considered the default
+export const AVAILABLE_WEIGHTS_PER_SIZE = {
+  500: ['bold'],
+  400: ['semibold', 'medium', 'bold'],
```

Comment:
```suggestion
  400: ['medium', 'semibold', 'bold'],
```

Reply:
See line 10: 
```
// notice: the first item in the array is considered the default
```

Reply:
Ok sorry I missed that.

Is there any way to arrange them in order of visual weight for the showcase?

Reply:
I've changed the logic as you suggested: https://github.com/hashicorp/design-system/pull/1490/commits/ce44d36ff4ffdab373d6bb1a435ad9123de7ca82